genesis-programs/src/lib.rs

```diff
@@ -76,7 +75,10 @@ pub fn get_programs(operating_mode: OperatingMode, epoch: Epoch) -> Option<Vec<(
         }
         OperatingMode::Preview => {
             if epoch == 0 {
-                Some(vec![solana_bpf_loader_program!()])
+                Some(vec![
+                    solana_bpf_loader_program!(),
```

Comment:
For all the three operating modes, the activation of `solana_bpf_loader_program!()` here needs a bit of work:

*  OperatingMode::Development -- 0 is ok for local testing.  But devnet is also moving forward so there also needs to be a future devnet epoch that also activates the new loader, that corresponds to shortly after when devnet is actually updated to this PR. You could use `epoch == std::u64::MAX` for now
* OperatingMode::Preview  -- needs to activate not at 0 but at some future epoch after all of testnet updates to this PR.  You could use `epoch == std::u64::MAX` for now
* OperatingMode::Stable -- Should be a `epoch == std::u64::MAX` case to activate 

Then we need three new issues, one for each operating mode and a new column in https://github.com/solana-labs/solana/projects/58 so that we can track the actual updating of devnet/testnet/mainnet-beta to include the new loader.